app/controllers/concerns/accessible.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+module Accessible
+  extend ActiveSupport::Concern
+
+  def execute_if_accessible(subject: nil, redirect: true)
+    return unless subject
```

Comment:
Also, semantically it should be `object`, since _subject_ (active participant) is always `current_user`.